platform/uwp/export/export.cpp

```diff
@@ -662,6 +663,27 @@ class EditorExportPlatformUWP : public EditorExportPlatform {
 			t++;
 		}
 
+		const char32_t *test_name = p_name.get_data();
+		int i = 0;
+		while (test_name[i]) {
+			if (!isalnum(test_name[i]) && '.' != test_name[i] && '-' != test_name[i]) {
+				return false;
+			}
+			i++;
+		}
```

Comment:
In this case it seems more natural to me to use a `for` loop instead of manually incrementing `i` from the outer scope:
```
for (int i = 0; test_name[i]; i++)
```

Might also be worth checking if `String::is_valid_identifier()` doesn't implement roughly what is done here to avoid redoing it.

Reply:
I can certainly switch to using a for loop.

Unfortunately `is_valid_identifier()` doesn’t fit the exact schema for the [Name](https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-identity) property, which allows for ‘-‘ and ‘.’ but otherwise only allows alphanumeric characters. We can switch so we don’t have any repeated code but I was just trying to make it as specific to the schema as possible.

Reply:
Yeah I suppose it makes sense to implement exactly what the MS docs specify.

Reply:
Okay the commits have been squashed and the requested amend was done! :)